packages/remix-react/links.ts

```diff
@@ -481,14 +505,14 @@ export function dedupe(descriptors: LinkDescriptor[], preloads: string[]) {
       return deduped;
     }
 
-    let str = JSON.stringify(descriptor);
-    if (!set.has(str)) {
-      set.add(str);
-      deduped.push(descriptor);
+    let key = JSON.stringify(sortKeys(descriptor));
```

Comment:
This fixes an unreported bug where the deduping didn't work correctly if properties were specified in a different order.